VAULT-32657 deprecate duplicate attributes in HCL configs and policies (#30386)
hashicorp/vault: website/content/partials/deprecation/duplicate-hcl-attributes.mdx

## Duplicate HCL attributes ((#duplicate-hcl-attributes))

| Announced | Expected end of support | Expected removal |
| :-------: | :---------------------: | :--------------: |
| JUN 2025  | OCT 2025                | N/A

> The usage of duplicate attributes in HCL configuration files and policy definitions is deprecated.

Reviewer: ```suggestion
The ability to duplicate attributes in HCL configuration files and policy definitions is deprecated.
```
Style correction: simply sentences when possible

> Look for "policy contains duplicate attributes" warnings in the server logs to identify if you are affected, once
> support for duplicate attributes ends any attempts to interact with such policies will result fail, like creating a
> token referencing it or even using a token that references it.

Reviewer: ```suggestion
To find affected policies, look for "policy contains duplicate attributes"
warnings in the server logs. Once we remove support for duplicate attributes,
you cannot interact with affected policies. For example, Vault will return an
error if you try to create a token referencing an affected policy or use an
existing token that references the policy.
```
Style correction: write in active voice, limit 1 idea per sentence, avoid ambiguous pronouns

> Configuration files provided to any Vault CLI commands are also subject to this deprecation and will warn about the
> usage of duplicate attributes.

Reviewer: ```suggestion
The Vault CLI now checks provided configuration files to determine if the file
is subject to the duplicate parameter deprecation and returns a warning about
the issue.
```
Style correction: write in active voice
